File: AULA_02/JAVA/OrdemCrescente.java
/* OrdemCrescente.java
 * Imprimir os 3 números em ordem crescente
 * Entrada: x,y,z (int)
 * Saída: números em ordem crescente
 *
 * Autor: Fabrício Olivetti de França
 * Disciplina Processamento da Informação
 * Universidade Federal do ABC
 */



import java.util.Scanner;

class OrdemCrescente{
	public static void main(String[] args) {
		int x,y,z;	
		Scanner leitor = new Scanner(System.in);

		// captura valores
		System.out.println("Entre com 3 números: ");
		x = leitor.nextInt();
		y = leitor.nextInt();
		z = leitor.nextInt();
		
    // x > y > z
    // x > z > y
    // y > x > z
    // y > z > y
    // z > x > y
    // z > y > x
    if( x < y && x < z){
      if( y < z ){
        System.out.println(x + " " + y + " " + z);
      }else{
        System.out.println(x + " " + z + " " + y);
      }
    }else if( y < z ){
      if( x < z ){
        System.out.println(y + " " + x + " " + z);
      }else{
        System.out.println(y + " " + z + " " + x);
      }
    }else{
      if( x < y ){
        System.out.println(z + " " + x + " " + y);
      }else{
        System.out.println(z + " " + y + " " + x);
      }
    }
	}
}

File: AULA_02/Python/OrdemCrescente.py
# -*- coding: cp1252 -*-
'''
OrdemCrescente.py
 * Imprimir os 3 números em ordem crescente
 * Entrada: x,y,z (int)
 * Saída: números em ordem crescente

Autor: Fabrício Olivetti de França
Disciplina Processamento da Informação
Universidade Federal do ABC
'''


x = float(raw_input("x = "))
y = float(raw_input("y = "))
z = float(raw_input("z = "))

if x<y and x<z:
    if y<z:
        print x,y,z
    else:
        print x,z,y
elif y<z:
    if x<z:
        print y,x,z
    else:
        print y,z,x
else:
    if x<y:
        print z,x,y
    else:
        print z,y,x
